doc/tutorial.txt

```diff
@@ -66,7 +66,7 @@ $ tree -d
 
 the configure step becomes
 
-./configure --host=aarch64-linux-gnu LDFLAGS="--static -L./../json-c/." CFLAGS="-I./.." --with-deadline
+./configure --host=aarch64-linux-gnu LDFLAGS="-static -L"`pwd`"/../json-c" CFLAGS="-I./../json-c" --with-deadline
```

Comment:
Please use $PWD instead of "`pwd`" it's easier to read and understand

Reply:
ok